addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -984,6 +984,164 @@ QUnit.module("Fields", (hooks) => {
         );
     });
 
+    /**
+     * When the user creates a property field of type many2many, many2one, etc.
+     * and changes the co-model of the field, the model loaded by the "Search more..."
+     * modal should correspond to the selected model and should be updated dynamically.
+     */
+    QUnit.test("properties: many2many 'Search more...' loaded model", async function (assert) {
+        async function mockRPC(route, { method, model }) {
+            if (method === "check_access_rights") {
+                return true;
+            }
+            if (method === "name_search") {
+                if (model === "res.partner") {
+                    return [
```

Comment:
Do we really need that many?  

If it's to trigger the search more to show up, maybe we could alter it and base the test on a search with 3 characters (see other comment).

Reply:
I will patch the `Many2XAutocomplete` component to change the default `searchLimit`.